src/main/connector/system/LinuxProcess/LinuxProcess.yaml

```diff
@@ -0,0 +1,53 @@
+extends:
+- ../System/System
+connector:
+  displayName: LinuxProcess
```

Comment:
It's a display name, so white spaces, etc. are allowed. Change to something like: `Linux - Processes (ps)` (we may have another implementation relying on `/proc` in the future)